app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  private
+
+  def current_user
+    User.where(id: session[:user_id]).first
```

Comment:
__[Optional]__ You don't have to use the `first` method here since the user id's are unique.